infrastructure/validators/onboarding/run-validator/secure-node.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Secure Your Validator
+description: Recommended practices to monitor, secure, and manage keys for your relay chain validator, along with some caveats on validators and hardware security
+---
+
+Validators in a Proof of Stake network are responsible for keeping the network in consensus and verifying state transitions. As the number of validators is limited, validators in the set have the responsibility to be online and faithfully execute their tasks.
+
+This primarily means that validators:
+
+- Must be high availability
+- Must have infrastructure that protects validators' signing keys so that an attacker cannot take control and commit [slash-able behavior](TODO:update-path){target=\_blank}
+
+## High Availability
+
+High availability set-ups that involve redundant validator nodes may seem attractive at first. However, they can be *very dangerous* if they aren't set up perfectly. The reason for this is that the session keys used by a validator should always be isolated to just a single node. Replicating session keys across multiple nodes could lead to equivocation [slashes](TODO:update-path){target=\_blank} or parachain validity slashes which can make you lose **100% of your staked funds**.
+
+The good news is that 100% uptime of your validator isn't really needed, as it has some buffer within eras to go offline for a little while and upgrade. For this reason, it is advised that you only attempt a high availability set-up if *you're confident you know exactly what you're doing.*
```

Comment:
i would say just do not do this, people should never do this

Reply:
Might be pertinent to note explicitly if you have two validators online with the same keys you **will** get slashed

Reply:
I think it would be better just to remove the whole section then? And make a note regarding the same keys being slashed

Reply:
@polkadot-developers/docs-content-team, can we please make a final decision on what needs to be removed, etc., and then I will review it for format?

Reply:
> I think it would be better just to remove the whole section then? And make a note regarding the same keys being slashed

Agree on that on my end